Index: .idea/aws.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"accountSettings\">\r\n    <option name=\"activeRegion\" value=\"us-east-1\" />\r\n    <option name=\"recentlyUsedRegions\">\r\n      <list>\r\n        <option value=\"us-east-1\" />\r\n      </list>\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/aws.xml b/.idea/aws.xml
--- a/.idea/aws.xml	(revision 2c5cf4a35b78e1484fdcbb13c34c2934be880bed)
+++ b/.idea/aws.xml	(date 1758312194138)
@@ -8,4 +8,7 @@
       </list>
     </option>
   </component>
+  <component name="connectionManager">
+    <option name="activeConnectionId" value="sso;us-east-1;https://view.awsapps.com/start" />
+  </component>
 </project>
\ No newline at end of file
